static/src/javascripts/projects/common/modules/email/run-checks.js

```diff
@@ -194,6 +204,7 @@ define([
         getEmailInserted: getEmailInserted,
         allEmailCanRun: allEmailCanRun,
         getUserEmailSubscriptions: getUserEmailSubscriptions,
-        listCanRun: listCanRun
+        listCanRun: listCanRun,
+        checkTestClash: checkTestClash
```

Comment:
I guess this is just for a test (thanks btw!) - can you differentiate it? I think we normally use a `_test` property that has an object of the test-only methods.